hack/helm_vars/wire-server/values.yaml.gotmpl

```diff
@@ -18,7 +18,7 @@ cassandra-migrations:
   imagePullPolicy: {{ .Values.imagePullPolicy }}
   cassandra:
     host: cassandra-ephemeral
-    replicaCount: 1
+    replicationFactor: 1
```

Comment:
aren't there any call sides that need to be changed in sync with this?

Reply:
The change in the previous PR is backwards compatible, so nothing *has* to be changed, but it's nice if it's changed so eventually the old stuff can be phased out.

There is https://github.com/wireapp/wire-server-deploy/pull/541

There are also references in cailleach, but those require a new version deployed to the respective environment first. I'll open separate PRs there.

Reply:
Created https://wearezeta.atlassian.net/browse/SQPIT-884 to track other deployments. Good concern!